_studio/shared/umc/core/umc/src/umc_video_data.cpp

```diff
@@ -293,7 +294,7 @@ Status VideoData::SetColorFormat(ColorFormat cFormat)
             align *= pFormat->m_PlaneFormatInfo[i].m_iAlignMult;
         }
         m_pPlaneData[i].m_nPitch =
-            align_value<size_t>(m_pPlaneData[i].m_ippSize.width * bpp, align);
+            mfx::align2_value<size_t>(m_pPlaneData[i].m_ippSize.width * bpp, align);
```

Comment:
I'm worrying about this one. It looks that ```align``` here could be not power of two

Reply:
Well, I think actual problem is
```
bpp = m_pPlaneData[i].m_iSampleSize * m_pPlaneData[i].m_iSamples;
align = MFX_MAX(m_iAlignment, bpp); 
```
because `m_iAlignMult` is either 2 or 1 for all supported `ColorFormatInfo`
`m_iAlignment` is not an issue because it is power of two (see `VideoData::SetAlignment`). A problem would be `bpp` which is `m_iSampleSize * m_iSamples`, for example for RGB24 `m_iSampleSize = (iBitDepth+7)>>3`  == 3. But `ColorFormatInfo` is defined w/ `m_iMinAlign` = 4 and we have `align = MFX_MAX(align, pFormat->m_iMinAlign)`

Anyway, we just improve (add assertion) behavior because old function works in the same way. I suggest to move on.

Reply:
Sure, I know that behavior here is not changed

Reply:
Do we still need explicit `size_t` here and above?

Reply:
no